popup.js

```diff
@@ -89,7 +89,6 @@ document.addEventListener("DOMContentLoaded", function() {
      download_btn.addEventListener("click",addDownload);
     addDownload();
```

Comment:
@rohancl Remove this function call as the event listener is calling it automatically with the argument, writing it here will make it run without the button press :slightly_smiling_face:

Reply:
^ Yeah @anuragparida  I just noticed the new code, BassaExtension automatically shows the notification without the buttonclick, because we called that function, I've removed the function call in the updated popup.js